### PR TITLE
Organization Account/profile Information Issue

### LINKED ISSUE #869: By default Org is created as 'I Need Help'
1. Open "https://staging.fightpandemics.work"
2. Navigate Profile -> Already Created Organizations 

Issue# You will observe that it display 'I need HELP' for the Organizations I have created by DEFAULT
Even can't edit the status for the org.

P.S refer GIF.
https://share.getcloudapp.com/X6uo8eZ5

### PR DESCRIPTION
**Pull request by tr1503**
### What does this pull request aim to achieve? 💯

Resolves #869  #878 

I removed "I need help/I want help" in the organization profile page for #869. 
For #878, the organization account information bug is fixed. Right now the checkboxes can display the needs part correctly.

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](https://github.com/FightPandemics/FightPandemics/blob/master/README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`** (need to have cloned the repo not forked).
- [x] This branch is rebased/merged with the **latest master**.
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".

### CODE REVIEW

File: client/.env [hunk 1]
```diff
@@ -1 +1,3 @@
 EXTEND_ESLINT=true
+REACT_APP_AIRTABLE_API_KEY="keyczjtMi7gM7IUKE" 
+REACT_APP_AIRTABLE_BASE="appi9N1EnrGen2B1g"
```
Comment (idjevm) [inline client/.env:3]: Please remove these are they are not needed on the frontend anymore. Also we try to avoid commiting secrets onto the repos.

Comment (tr1503) [reply client/.env]: Got it! I just removed it and pushed the updated version.

**Comment by idjevm** [comment]
hey @tr1503 I will close this up since it is starting to get so messy already, so please open a new one with your branch named `feature/<branch-name>`. Ideally, for these changes, we would want to allow other to review in a deployed version of this branch, by having `feature/` at the beginning of your branch, this should work well. Please make sure you do commit something so that way the HEAD hash will be different. Also, there is no need to do all these merges, so maybe just create a new branch with the suggested naming, and just put your changes there, open a new PR and request my review 🙏

**Comment by vikjoshi** [comment]
@tr1503 looks like Im still seeing these issues in staging- guessing thats because of @idjevm s comment above. Once you create a new branch and get JVs review, please let me know so I can quickly test and close out this issue. Thank you!

**Comment by idjevm** [comment]
Yeah this has not been merged @vikjoshi , however there is a new PR #980, which seems to be working fine. Can you please confirm it on the build URL?

**Pull request closed without merging**